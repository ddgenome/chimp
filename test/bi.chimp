# bimolecular surface reaction
mechanism "bi.mech"
## parameter input
parameter "bi.par"
## simple task
task "bi.task"
