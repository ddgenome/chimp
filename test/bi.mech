# bimolecular surface reaction mechanism
# adsorption / desorption
A + @ -> k(A_Aads) @A;
# surface reaction
2 @A -> k_arrhenius(A_sr, E_sr) B + 2@;
