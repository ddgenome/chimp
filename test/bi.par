# parameter input file for simple mechanism
A_Aads	1.0e1	# molec/Pa/site/s
A_sr	2.0e9	# molec/site/s
E_sr	5.7e4	# J/mol
