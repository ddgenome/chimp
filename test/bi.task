# -*- text -*-
# bi input
begin model bi
  output "bi.out"
  begin integrator kmc
    size 30
    begin state
      begin quantity
	p[A] = 1.0e5		# Pa
	@[@A] = 1.0e0
      end quantity
      begin output
	1.0e-5 1.0e-1 1.0e0 2.0e0
      end output
      begin reactor batch
	temperature 3.5e2	# K
	pressure 1.0e5		# Pa
	volume 5.0e-4		# m^3
	sites 9.0e18
	rate_numerator molecules
	rate_denominator sites
	fluid_quantity pressure
      end reactor
    end state
  end integrator
end model
