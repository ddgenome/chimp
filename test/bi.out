# bi
# x	@	@A	A	B	steps
0.000000e+00	1.976197e-14	1.000000e-00	1.000000e+05	0.000000e+00	0
4.353186e-04	2.222222e-03	9.977778e-01	1.000000e+05	9.664599e-02	1
1.000139e-01	2.222222e-03	9.977778e-01	9.989543e+04	5.238213e+01	1624
1.000210e+00	2.222222e-03	9.977778e-01	9.893515e+04	5.325194e+02	16528
2.000385e+00	2.222222e-03	9.977778e-01	9.782566e+04	1.087267e+03	33748
